r_markdown/emissions.Rmd

```diff
@@ -442,13 +548,6 @@ ___
   ions <- dplyr::mutate(ions, 
                         mass_emitted = (conc - conc_bg) * flow_hood * dur,
                         mass_carbon = mass_emitted * 0)
-
-  ions_g <- dplyr::group_by(ions, id) %>%
```

Comment:
Hey! Why did this get deleted? Also, I was wondering if we should add an ions_ or carbonyls_ before the names of those pollutants in the emissions specific Rmardown. That would make it convenient to calculate total pollutants...